opendatahub/docs/get-started-odh-modelserving.md

```diff
@@ -1,29 +1,30 @@
-# Getting Started
+# Getting Started with ODH ModelMesh Serving
 
-## Prerequisites
+The provided deploy script allows you to quickly run OpenDataHub ModelMesh Serving with a provisioned `etcd` server. This deploy script is intended for experimentation or development purposes only. It is not intended for production purposes.
 
-- **OpenShift cluster** - A OpenShift cluster is required. You will need `cluster-admin` or `dedicated-admin` authority in order to complete all of the prescribed steps.
+## Prerequisites
 
-- **OpenShift cli** - The installation will occur via the terminal using [oc](https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands).
+- You have `cluster-admin` or `dedicated-admin` access to an OpenShift cluster.
```

Comment:
Just found that `or dedicated-admin` needed to be removed. please remove this part.